tests/TestUTestMacro.cpp

```diff
@@ -1222,7 +1225,7 @@ IGNORE_TEST(UnitTestMacros, BITS_EQUAL_TEXTWorksInAnIgnoredTest)
 } // LCOV_EXCL_LINE
 
 #if CPPUTEST_USE_STD_CPP_LIB
-static void _failingTestMethod_NoThrowWithCHECK_THROWS()
+static void failingTestMethod_NoThrowWithCHECK_THROWS_()
```

Comment:
Moved it from front to back. Just to trigger errored Travis CI. But then, in many other places, it's a postfix not a prefix, so there is some sense in it :)